vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -6,26 +6,8 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
-          "1.22.11-hotfix.20221109.1",
```

Comment:
lol we were caching all of these?

Reply:
yes, we were. But 1.23 just recently got deprecated